database/factories/BeatmapDiscussionFactory.php

```diff
@@ -26,6 +27,13 @@ class BeatmapDiscussionFactory extends Factory
 
     protected $model = BeatmapDiscussion::class;
 
+    public function configure()
+    {
+        return $this->afterCreating(function (BeatmapDiscussion $discussion) {
+            $discussion->wasRecentlyCreated = false;
```

Comment:
can use a comment on why this is needed (and seems like something that should be its own state instead of always applied just for this one model?)

(or maybe just `->fresh()` right after create)

Reply:
or just stop relying on `wasRecentlyCreated` as it kept resulting in obtuse behaviour when I was moving things around ಠ_ಠ